cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -67,6 +67,9 @@ message CassandraFrameworkConfiguration {
      * target number of seed nodes.
      */
     optional int32 targetNumberOfSeeds = 9;
+
+    /** Default rack/dc info */
+    optional RackDc defaultRackDc = 10;
```

Comment:
Once this has been moved, please also update [io.mesosphere.mesos.frameworks.cassandra.scheduler.util.JaxRsUtils#writeConfigRole](https://github.com/mesosphere/cassandra-mesos/blob/master/cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/util/JaxRsUtils.java#L155) to write the default rack and dc.